solver/commonSolver.py

```diff
@@ -292,11 +292,8 @@ def removeItemAt(self, locNameWeb):
         # item
         item = loc.itemName
 
-        if self.mode in ['seedless', 'race', 'debug']:
```

Comment:
Ah, iirc this was during the state reset when updating from an autotracking game state.  Somewhere this function got called along the way, and it was attempting to remove the Nth collected item if we were clearing the Nth location, because it's assuming the collected items directly correspond to the visited locations in the same order.

I could add a flag of some kind when resetting during an import dump if we're reading inventory from memory?